File: C/Atividade/T3/1.py
chico:float    = 1.5
ze:float       = 1.1
cr_chico:float = 0.02
cr_ze:float    = 0.03
c:int          = 0

while chico > ze:
    chico = chico + cr_chico
    ze = ze + cr_ze
    c += 1
print(f"Serão Precisos {c} anos para Zé > Chico.")
    #if chico < ze:
        #print("STOP!!!!!!!!!!!!!!")

File: C/Atividade/T3/1.c
#include <stdio.h>
#include <stdlib.h>

int main(){
    float chico = 1.5, ze = 1.1;
    float cr_chico = 0.02, cr_ze = 0.03;
    int c;

    for (c = 0; chico > ze; ++c) {
        chico = chico + cr_chico;
        ze = ze + cr_ze;
    }
    printf("Serão Precisos %i anos para Zé > Chico.", c);
    return 0;
}